test/org/zaproxy/zap/extension/openapi/HTTPDTestServer.java

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Is keeping extension specific versions of these files going to bite us in the behind down the road? 

Can we genericize `HTTPDTestServer` and `NanoServerHandler`?